services/search/testing/test_search_engine.py

```diff
@@ -9,8 +11,13 @@ def test_same_weights_for_all():
     """
     result = {1: 3}
 
-    assert result == SearchEngine().search_query(constants.KEYWORDS_ARTICLE_1,
-                                                 constants.SYNONYMS_ARTICLE_1)
+    with mock.patch('requests.post', return_value=constants.KEYWORDS_DB_MOCK_1) as mocked:
+        with mock.patch('env.get_db_endpoint', return_value="http://somerealurl.com"):
```

Comment:
it would make the test cleaner to extract the mock URL to a variable and reuse it in the assert